src/server/status/index.js

```diff
@@ -29,7 +34,8 @@ export default function (kbnServer, server, config) {
           build_snapshot: matchSnapshot.test(config.get('pkg.version'))
         },
         status: kbnServer.status.toJSON(),
-        metrics: kbnServer.metrics
+        metrics: kbnServer.metrics,
+        stats,
```

Comment:
This is exactly what I had in mind. Part of me wonders if we should create a new `_kibana/stats` endpoint in Kibana, analogous to `_nodes/stats` for a specific Elasticsearch node, and start introducing this type of functionality.

At that point `stats` should could be either renamed or merged entirely with the top-level object. Then we could also cleanup the current API _in that new API_ in terms of what it returns as it's a mix of "info" and "stats" even though it purports to be "status" only.